js/app.js

```diff
@@ -13,15 +13,27 @@ fav.forEach(wishI => wishI.addEventListener('click', (event) => {
     event.target.classList.toggle("fa-heart-o", !isFavProduct)
 
 }));
-
+const cartItem = document.querySelector(".cart")
 const btns = document.querySelectorAll('.btn');
 const cart = new Cart();
 btns.forEach((button)=>{
     button.addEventListener('click',(e)=>{
         e.preventDefault()
+        console.log(e.target);
         const id = getProductId(e.target);
         cart.addToCart(id)
         console.log(cart)
         console.log(getProductById(id))
+        console.log(getProductById(id).image);
+        cartItem.innerHTML+=`
+        <div class="row border-top border-bottom">
```

Comment:
Majd azt is old meg, hogy ne tobbszor keruljon be ugyan az az elem, hanem csak egyszer nagyobb `qty`-val